src/pages/VaccineModulePage.js

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable react/forbid-prop-types, global-require */
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2019
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Image, StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { Button } from 'react-native-ui-components';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import { VaccineChart } from '../widgets/VaccineChart';
+import globalStyles, {
+  SHADOW_BORDER,
+  APP_FONT_FAMILY,
+  SUSSOL_ORANGE,
+  GREY,
+  HAZARD_RED,
+} from '../globalStyles';
+
+const CHERVON_ICON_STYLE = { size: 18, color: SUSSOL_ORANGE };
+const BREACH_ICON_STYLE = { size: 25, color: HAZARD_RED };
+
+// TODO navigation for menu buttons
```

Comment:
Should rebase `vaccine-development` before this - new navigation has come in hot